sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -213,16 +217,16 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
       this._logger.verbose(
         `${this._logPrefix} Lock ${this._openLock} acquired for initializing link`
       );
-      return this._initLinkImpl(options, abortSignal);
+      return this._initLinkImpl(options, operationOptions);
     });
   }
 
   private async _initLinkImpl(
     options: LinkOptionsT<LinkT>,
-    abortSignal?: AbortSignalLike
+    operationOptions: OperationOptionsBase = {}
```

Comment:
You have not made this _worse_ but we might want to consider making some of these parameters non-optional (ie, you must explicitly pass undefined or something like that) just to make it harder to not propagate the abortSignal or other parameters.

Reply:
Yes